src/sections/error-page/ErrorPage.tsx

```diff
@@ -11,6 +11,11 @@ export function ErrorPage() {
   useErrorLogger(error)
   const theme = useTheme()
 
+  const header = document.querySelector('nav')
+  const footer = document.querySelector('footer')
+  const newMinHeight = header && footer ? '$main-container-available-height' : '100vh'
+  document.documentElement.style.setProperty('--error-min-height', newMinHeight)
+
```

Comment:
I would avoid running natives javascript dom selectors to find elements by their tag name or even by a class name, these might change and might not even be the only ones you find.

Take a look at this, is from my lazy imports PR, see [this tsx file](https://github.com/IQSS/dataverse-frontend/blob/09edc74271179dfd5b5ef185cc25d3143be68512/src/sections/shared/layout/app-loader/AppLoader.tsx) and [this scss one](https://github.com/IQSS/dataverse-frontend/blob/09edc74271179dfd5b5ef185cc25d3143be68512/src/sections/shared/layout/app-loader/AppLoader.module.scss).

So then you only need to pass that prop the ErrorPage on the root layout path object.
```

  {
    path: '/',
    element: <Layout />,
    errorElement: <ErrorPage fullViewport />,
    children: [
    ...rest
```

Reply:
OK, I'll follow the example. Thank you!